operator/builtin/input/journald/journald.go

```diff
@@ -163,9 +167,6 @@ func (operator *JournaldInput) startPoller(ctx context.Context) {
 
 // poll checks all the watched paths for new entries
 func (operator *JournaldInput) poll(ctx context.Context) error {
-	operator.wg.Add(1)
-
-	defer operator.wg.Done()
```

Comment:
Is this the superfluous call? Isn't it useful in that it the poll method will have to complete before the wait group is decremented?